vmm/sandbox/src/stratovirt/hooks.rs

```diff
@@ -42,8 +42,9 @@ impl Hooks<StratoVirtVM> for StratoVirtHooks {
     }
 
     async fn post_start(&self, sandbox: &mut KuasarSandbox<StratoVirtVM>) -> Result<()> {
-        let data = &mut sandbox.data;
-        data.task_address = sandbox.vm.agent_socket.to_string();
+        sandbox.data.task_address = sandbox.vm.agent_socket.to_string();
+        // sync clock
+        sandbox.sync_clock().await;
```

Comment:
the clocking syncing seems hypervisor agnostic, so maybe it better calling it after client is initialized, rather than calling it in every post_start method

Reply:
Updated